_info/Forge

```diff
@@ -1,3 +1,5 @@
 Forge
 
-The main source of income this game is called the forge. Every time a phase changed, every member of your team will get 100*<Your forge level> amount of cash. The forge level can be accessed by using `!disforge` and you can use `!upforge` to upgrade it! Doing this is very useful in the long run. Note that solo roles get a level 7 forge from the start.
+The main source of income this game is called the forge. Every time a phase changed, every member of your team will get 100*<Your forge level> amount of cash. 
```

Comment:
"The main source of income this game is called the forge" -> "You teams main source of income comes from the forge" or something along those lines

Reply:
Fixed I think